noxfile.py

```diff
@@ -755,23 +763,56 @@ def notebook(session: nox.Session):
             CURRENT_DIRECTORY / "scripts" / "notebooks_fill_params.py",
             *notebooks,
         )
+        processes = []
+        manager = multiprocessing.Manager()
+        error_flag = manager.Value("i", False)
+
         for notebook in notebooks:
-            session.run(
+            args = (
                 "python",
                 "scripts/run_and_publish_benchmark.py",
                 "--notebook",
                 f"--benchmark-path={notebook}",
             )
+            if multi_process_mode:
+                process = multiprocessing.Process(
+                    target=_run_process,
+                    args=(session, args, error_flag),
+                )
+                process.start()
+                processes.append(process)
+            else:
+                session.run(*args)
+
+        for process in processes:
```

Comment:
[nit] clear processes list as now we are referring to benchmark notebook processes